src/fibex2som.rs

```diff
@@ -1149,6 +1167,68 @@ mod builder {
             _ => SOMTypeField::U32,
         }
     }
+
+    fn get_bitfield_type_items(type_decls: &[FibexTypeDeclaration]) -> Vec<SOMBitfieldMember> {
+        let mut items = Vec::new();
+
+        for type_decl in type_decls {
+            if type_decl.is_array() {
+                continue;
+            }
+
+            if let Some(bit_len) = type_decl.get_bit_length() {
+                if let Some(type_ref) = &type_decl.type_ref {
+                    let type_def = type_ref.borrow();
+
+                    let name = type_decl.name.clone();
+                    let description = type_def.name.clone();
+
+                    let datatype = match &type_def.datatype {
+                        FibexDatatype::Primitive(datatype) => datatype,
+                        FibexDatatype::Enum(datatype) => &datatype.primitive,
+                        _ => {
```

Comment:
better not use the global `_` pattern since it will reduce type safety

Reply:
An if-let-else could be used, but it would evaluate to the same match statement according: https://rust-lang.github.io/rfcs/0160-if-let.html. For the sake of readability the match is prefered here and below.